src/kernel/src/process/module/recompiler.rs

```diff
@@ -790,7 +803,7 @@ impl<'input> Recompiler<'input> {
     }
 
     fn transform_ud2(&mut self, i: Instruction) -> usize {
-        let handler: extern "sysv64" fn(&mut Process, usize) -> ! = Process::handle_ud2;
+        let handler: extern "C" fn(&mut Process, usize) -> ! = Process::handle_ud2;
```

Comment:
We also need to fix the code that invoke this on Windows because it uses different calling conventions.

Reply:
Sysv64 is for non-windows x86_64 but it still compiles on windows?

Reply:
To solve this we could change the sysv64 to x86_64 only and use ```extern C``` for arm as there's no equivalent call convention.